### PR TITLE
fix: Allow sending distinctID and fix bug with active flags

### PR DESCRIPTION
**Pull request by neilkakkar**
## Problem

Some bug fixes on node

### Bump level

- [ ] Major
- [ ] Minor
- [x] Patch

### Libraries affected

- [ ] All of them
- [ ] posthog-web
- [x] posthog-node
- [ ] posthog-react-native

### Changelog notes

- Added support for X

### CODE REVIEW

File: posthog-node/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 2.5.3 - 2023-02-21
+
+1. Allow passing in a distinctId to `groupIdentify()`.
+2. Fix a bug with active feature flags on capture events, where non-active flags would be added to the list as well.
```
Comment (liyiy) [inline posthog-node/CHANGELOG.md:4]: can we add a lil test for this too? 🥺 For consistency across libraries and test spec documentation

**Review by liyiy — APPROVED** [review]

File: posthog-node/CHANGELOG.md (hunk 1 above)
Comment (neilkakkar) [reply posthog-node/CHANGELOG.md:4]: I did, that's what the 'disabled_flag: false' is doing in tests

Comment (neilkakkar) [reply posthog-node/CHANGELOG.md:4]: (so I effectively added two tests 😂 )

Comment (liyiy) [reply posthog-node/CHANGELOG.md:4]: oops didn't see originally it but i guess that works LOL :P

**Pull request merged**